lineamigos/urls.py

```diff
@@ -13,8 +13,11 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
+   path('/user', include('user.urls')),
+    path('/product',include('product.urls')),
+    path('/review',include('review.urls')),
+    path('/order',include('order.urls'))
```

Comment:
main url에는 슬래시('/')를 붙이지 않고, app url의 앞부분에 슬래시를 붙이시는 것이 위코드 규칙입니다 :)
```suggestion
   path('user', include('user.urls')),
    path('product',include('product.urls')),
    path('review',include('review.urls')),
    path('order',include('order.urls'))
```

Reply:
수정하겠습니다.